eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -6,6 +6,21 @@ jobs:
   condition: and(succeededOrFailed(), ne(variables['Skip.Test'], 'true'))
   strategy:
     matrix:
+      Linux_ARM:
+        OSVmImage: 'ubuntu-18.04'
+        vcpkg.deps: ''
+        VCPKG_DEFAULT_TRIPLET: 'x64-linux'
```

Comment:
I would expect arm64-linux as the triplet

Reply:
I've removed the triplet because it's not necessary (we aren't using vcpkg)